app/models/calendar.rb

```diff
@@ -18,4 +19,9 @@ def new_ms_schedule weekdays
         false
     end
   end
+
+  def update_last_speech
+    last_calendar = user.calendars.day_desc.first
+    user.update! last_speech: last_calendar&.day
```

Comment:
c thấy e có bắt exception đâu mà dùng !
dùng update_attribute kia an toàn hơn vì e k dùng transaction. nó chỉ cập nhập trường kia thôi bỏ qua validate

Reply:
vâng ạ.